tmt/export.py

```diff
@@ -497,6 +509,57 @@ def export_to_nitrate(test):
             raise ConvertError("Couldn't update bugs", original=err)
 
 
+def add_to_nitrate_release(nitrate_case, general_plan, test, dry_mode):
+    """
+    Go down plan tree from general plan, add case and case run to all open runs.
+
+    Try to apply adjust
+    """
+    for child_plan in nitrate.TestPlan.search(parent=general_plan.id):
+        for testrun in child_plan.testruns:
+            if testrun.status == nitrate.RunStatus("FINISHED"):
+                continue
+            if not enabled_for_environment(test, tcms_notes=testrun.notes):
+                continue
+            if not nitrate_case or child_plan not in nitrate_case.testplans:
```

Comment:
Thinking about the `if not nitrate_case` condition: Here `nitrate_case` can be `None` as well? Line 527 suggests that not.

Reply:
nitrate_case is None for drymode

Reply:
So... for `dry` mode the `Link to plan...` message will always be shown? Even if already included?

Reply:
No. It will be printed only if
a) --create would create new test case or
b) plan is not yet included.

Or did I mess up the logic? 
The nitrate_case is kept as  None around https://github.com/teemtee/tmt/blob/lzachar-export-release/tmt/export.py#L273

Reply:
Ah, I see, so the `not nitrate_case` condition covers when `--create --dry` are used together. Makes sense. Thanks for clarifying.